app/views/sorns/search.html.erb

```diff
@@ -63,6 +63,10 @@
             <span class="csv-download-link-text">Download results as a CSV file</span>
           <% end %>
         </div>
+        <div id="about">
+          <p><a href="https://all-sorns.app.cloud.gov/about">About this dashboard</a></p>
```

Comment:
To have this link work locally and the dev servers, we'll use rails link_to method. 
```suggestion
          <p><%= link_to 'About this dashboard', about_path %></p>
```

Reply:
Rails creates that `about_path` for us, based on the methods available in our controllers. You can go to http://localhost:3000/rails/info/routes to see all the available paths and routes. 

Now that I've done that, I see we've left the CRUD routes available for SORNs. No good, I'll make a ticket to fix that.